### ISSUE TITLE
Cell.text property incorrect when using format cells

### ISSUE THREAD

**Issue by vivet** [EPPlusSoftware/EPPlus#201]
Hi

When I have cells formatted The Cell.Text property doesn't follow that.
I have attached a simple sheet, demonstrating this.

```
Cells["A1"].Text  // This outputs "-", but should output "0" (as seen in excel)
Cells["A2"].Text  // This outputs correct.
Cells["A3"].Text  // This outputs correct.
```
[Book1.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5421031/Book1.xlsx)

Let me know if you need more information. 
Thanks for a great product.

**Comment by JanKallman** [comment]
We'll have a look

**Comment by JanKallman** [comment]
Hello,
This is a difference between .NET and Excel when formatting numbers.
.NET seems to format using the rounded value:
`(0.01).ToString("#,##0;-#,##0;-") --> "-"`, since it it rounds 0.01 to zero before formatting. 
Excel doesn't round before formatting.
`(0.01).ToString("#,##0;-#,##0;-") --> "0"`

We'll try to provide a fix, but it might take some time.

**Comment by vivet** [comment]
Hi

Thanks for the quick reply.
Looking forward to a fix.

**Comment by JanKallman** [comment]
Fixed in 5.4.1

**Comment by vivet** [comment]
Thanks
